app/src/main/java/fnsb/macstransit/MapsActivity.java

```diff
@@ -2,39 +2,35 @@
 
```

Comment:
This file is huge. Should really focus on trimming it down.

Perhaps move most of the non-essential methods to other classes...